.github/workflows/docker.yml

```diff
@@ -19,19 +23,12 @@ jobs:
       - name: Install dependencies
         run: yarn install --immutable --immutable-cache
 
+      - name: build openapi
+        run: yarn workspace openapi build
+
       - name: Build Docker Image
         run: yarn docker build backend -t ghcr.io/${{ github.repository_owner }}/rafiki-backend:${{ github.sha }} -t ghcr.io/${{ github.repository_owner }}/rafiki-backend:latest
         
       - name: Tag Docker Image with Tag Name
         if: github.ref_type == 'tag'
         run: docker tag ghcr.io/${{ github.repository_owner }}/rafiki-backend:${{ github.sha }} ghcr.io/${{ github.repository_owner }}/rafiki-backend:${{ github.ref_name }}
-
```

Comment:
I removed these steps because I did not want runs on this branch to actually publish docker images to our repos. I will replace them before merging.